### PR TITLE
Fix compatiblity issues with Python 3.8

### PR DESCRIPTION
**Pull request by masipcat**

### CODE REVIEW

File: guillotina/tests/test_api.py [hunk 1]
```diff
@@ -152,8 +152,7 @@ def uninstall(cls, container, request):
         assert status == 201
         _, status = await requester("GET", "/db/guillotina/item1")
         assert _["@static_behaviors"][0] == "guillotina.behaviors.dublincore.IDublinCore"
-        request = utils.get_mocked_request(db=requester.db)
-        root = await utils.get_root(request)
```
Comment (masipcat) [inline guillotina/tests/test_api.py]: I don't understand why this wasn't failing with python 3.7...

File: guillotina/tests/test_postgres.py [hunk 2]
```diff
@@ -507,10 +506,10 @@
     with TransactionManager(aps) as tm, await tm.begin() as txn:
         await txn.get_connection()
 
-        with pytest.raises(concurrent.futures._base.TimeoutError):
+        with pytest.raises(asyncio.exceptions.TimeoutError):
             # should throw an error because we've run out of connections in pool
             txn2 = await tm.begin()
-            await asyncio.wait_for(txn2.get_connection(), 0.5)
```
Comment (masipcat) [inline guillotina/tests/test_postgres.py]: `get_connection()` calls `storage.open()` that already has a timeout

File: guillotina/tests/test_postgres.py [hunk 3]
```diff
@@ -507,10 +506,10 @@
     with TransactionManager(aps) as tm, await tm.begin() as txn:
         await txn.get_connection()
 
-        with pytest.raises(concurrent.futures._base.TimeoutError):
+        with pytest.raises(asyncio.exceptions.TimeoutError):
```
Comment (masipcat) [inline guillotina/tests/test_postgres.py]: https://github.com/python/cpython/pull/9141

**Comment by vangheem** [comment]
@masipcat oh nice, I didn't know you could do that. Looks good to me.

**Comment by vangheem** [comment]
Thank you!

**Pull request merged**